src/utils/authentication-util.ts

```diff
@@ -1,27 +1,41 @@
+import Cookies from "js-cookie";
+
 export class AuthenticationUtil {
   private static readonly USER = "user";
   private static readonly USER_TOKEN = "user-token";
 
+  // Attributes for cookies
+  private static readonly ATTRS = {
+    expires: 1,
+    secure: true,
+    samesite: "lax",
```

Comment:
there's a httpOnly but I guess that's not useful as it doesn't get sent with XHR requests then, right?

Reply:
We do not have any XHR requests, so we do not need this attribute

Reply:
We don't? Then we should set the httpOnly to true.